packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard.patient.spec.tsx

```diff
@@ -48,8 +51,8 @@ import { type AppMainLayoutParams, testAppMainLayoutForPatient } from '../../use
 import { type PatientDashboardLayoutParams } from '../../assert/layout.assert'
 import {
   testDashboardDataVisualisationForPatient,
-  testDashboardDataVisualisationPrivateTeamNoData,
-  testDashboardDataVisualisationWithTwoWeeksOldData,
+  testDashboardDataVisualisationPrivateTeamNoData, testDashboardDataVisualisationTwoWeeksOldData,
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏This import should be aligned with the others